sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java

```diff
@@ -206,4 +207,38 @@ public void createSynchronousServiceBusDeadLetterQueueReceiver() {
             .subQueue(SubQueue.DEAD_LETTER_QUEUE)
             .buildClient();
     }
+
+    /**
+     * Code sample for creating a Service Bus Processor Client.
+     */
+    public void createServiceBusProcessorClient() {
+        // Sample code that processes a single message
+        Consumer<ServiceBusReceivedMessageContext> processMessage = messageContext -> {
+            try {
+                System.out.println(messageContext.getMessage().getMessageId());
+                // other message processing code
+                messageContext.complete();
+            } catch (Exception ex) {
+                messageContext.abandon();
+            }
+        };
+
+        // Sample code that gets called if there's an error
+        Consumer<ServiceBusErrorContext> processError = errorContext -> {
+            System.err.println("Error occurred while receiving message: " + errorContext.getException());
+        };
+
+        // create the processor client via the builder and its sub-builder
+        ServiceBusProcessorClient processorClient = new ServiceBusClientBuilder()
+                                        .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
+                                        .processor()
+                                        .queueName("<< QUEUE NAME >>")
+                                        .processMessage(processMessage)
```

Comment:
Add `.disableAutoComplete()`

By default `autoComplete` is enabled,  this means this SDK will internally  `complete` the message for you and user do not need to call `messageContext.complete()` .
But since we want to show how  user can `disable this feature`  by calling `disableAutoComplete` and complete it manually (like you do in  `processMessage` Consumer).

We had this bug in readme and it was  left over,  since you are touching this specific part, you can fix this also.  Thank You

Reply:
I used `disableAutoComplete` in a project to fix exactly the issue you wrote. 
I added the change to the pull request.